spec/requests/jobseekers/job_applications_spec.rb

```diff
@@ -0,0 +1,19 @@
+require "rails_helper"
+
+RSpec.describe "Job applications", type: :request do
+  let(:vacancy) { create(:vacancy, organisation_vacancies_attributes: [{ organisation: build(:school) }]) }
+  let(:jobseeker) { create(:jobseeker) }
+
+  before do
+    allow(JobseekerApplicationsFeature).to receive(:enabled?).and_return(true)
+    sign_in jobseeker
```

Comment:
Where does this come from? In their docs I only ever saw `login_as(jobseeker, scope: :jobseeker)`

Reply:
https://github.com/heartcombo/devise/wiki/How-To:-sign-in-and-out-a-user-in-Request-type-specs-(specs-tagged-with-type:-:request)